diff -urN gnubiff-2.2.15/src/Makefile.in gnubiff-2.2.15-patched/src/Makefile.in
--- gnubiff-2.2.15/src/Makefile.in	2012-01-29 19:17:28.000000000 +0100
+++ gnubiff-2.2.15-patched/src/Makefile.in	2012-06-13 16:36:39.191025903 +0200
@@ -755,7 +755,7 @@
 gnubiff$(EXEEXT): $(gnubiff_OBJECTS) $(gnubiff_DEPENDENCIES) 
 	@rm -f gnubiff$(EXEEXT)
 	@printf "%-40s" "Linking $@ ... "
-	$(CXXLINK) $(gnubiff_LDFLAGS) $(gnubiff_OBJECTS) $(gnubiff_LDADD) $(LIBS)
+	$(CXXLINK) $(gnubiff_LDFLAGS) $(gnubiff_OBJECTS) $(gnubiff_LDADD) $(LIBS) -lgthread-2.0
 	@echo "[OK]"
 
 $(panelapplet_in_files): $(panelapplet_in_files:.panel-applet.in=.panel-applet.in.in)
